### DISCUSSION TITLE
To skip page numbers for 'outline()` of arbitrary length

### DISCUSSION THREAD

**Discussion by johnmapeson** [typst/typst#4955] [Q&A] [answered]
If I know that `outline()` will take only one page, and I want it to not have a page number, I can use `counter(page).update(2)`:

```
#outline()
#pagebreak()

#set page(numbering: "1")
#counter(page).update(2)

#lorem(100)
```

Likewise, if I know that `outline()` will take two pages, and I want them to not have a page number, I will use `counter(page).update(3)`:

```
#outline()
#pagebreak()
#pagebreak() // The second pagebreak is added simply because outline is currently empty

#set page(numbering: "1")
#counter(page).update(3)

#lorem(100)
```

But how to make it work if I don't know how many pages the outline will take? Maybe just one page, maybe two, maybe three etc. That is, if the `outline()` will take, for example, five pages, the numbering should start from the sixth page, and it should be numbered "6".

I tried things like `#counter(page).update(counter(page).self.get)`, but got errors.

**Comment by laurmaedje** [comment, marked as answer]
Maybe I misunderstood, but this should be the default behaviour. Just don't add any updates:
```typ
#outline()
#pagebreak()
#pagebreak() // The second pagebreak is added simply because outline is currently empty

#set page(numbering: "1")

#lorem(100)
```

**Reply by johnmapeson** [reply]
Thanks. Yes, this is a thing I didn't try, for some reason.
